src/uwtools/resources/jsonschema/execution.jsonschema

```diff
@@ -0,0 +1,34 @@
+{
+  "additionalProperties": false,
+  "properties": {
+    "batchargs": {
+      "type": "object"
+    },
+    "envcmds": {
+      "items": {
+        "type": "string"
+      },
+      "type": "array"
+    },
+    "executable": {
+      "type": "string"
+    },
+    "mpiargs": {
+      "items": {
+        "type": "string"
+      },
+      "type": "array"
+    },
+    "mpicmd": {
+      "type": "string"
+    },
+    "threads": {
+      "minimum": 0,
+      "type": "integer"
+    }
+  },
+  "required": [
+    "executable"
+  ],
+  "type": "object"
+}
```

Comment:
The schema for the `execution` block common to both the `fv3` and `sfc_climo_gen` configs.